inc/testrunnerswitcher.h

```diff
@@ -54,6 +58,8 @@ typedef void* TEST_MUTEX_HANDLE;
 #define TEST_MUTEX_RELEASE(mutex)
 #define TEST_MUTEX_DESTROY(mutex)
 
+// This code belongs in ctest, there is an issue files to move it there:
+// https://github.com/Azure/azure-c-testrunnerswitcher/issues/32
```

Comment:
why put this comment in the code?  Why don't you just move it instead of saying that you're going to move it.